hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestBlockOutputStreamWithFailures.java

```diff
@@ -501,14 +501,14 @@ public void test2DatanodesFailure() throws Exception {
     // and one flush for partial chunk
     key.flush();
 
+    Throwable ioException = HddsClientUtils.checkForException(
+        blockOutputStream.getIoException());
     // Since, 2 datanodes went down, if the pipeline gets destroyed quickly,
     // it will hit GroupMismatchException else, it will fail with
     // RaftRetryFailureException
```

Comment:
Please update the comment stating it can also fail with container not open exception in case the containers got closed quickly.